comprobo_final_project/scripts/simulator/simulation_visualizer.py

```diff
@@ -32,11 +43,31 @@ def render(self):
         axes.set_ylim([-10, 10])
 
         # http://matplotlib.org/api/pyplot_api.html#matplotlib.pyplot.quiver
-        self.quiver_manager = axes.quiver(
-            self.robot.pose.position.x,
-            self.robot.pose.position.y,
-            self.robot.pose.velocity.x,
-            self.robot.pose.velocity.y,
+        self.quiver_manager1 = axes.quiver(
```

Comment:
We don't need all these quiver managers. A quiver manager can plot multiple quivers.